docs/hazmat/primitives/asymmetric/serialization.rst

```diff
@@ -459,6 +459,187 @@ An example ECDSA key in OpenSSH format::
     :raises cryptography.exceptions.UnsupportedAlgorithm: If the serialized
         key is of a type that is not supported.
 
+
+OpenSSH Certificate
+~~~~~~~~~~~~~~~~~~~
+
+The format used by OpenSSH for certificates, as specified in
+`PROTOCOL.certkeys`_.
+
+.. function:: load_ssh_certificate(data)
+
+    .. versionadded:: 40.0
+
+    .. note::
+
+        This function does not support parsing certificates with DSA public
+        keys or signatures from DSA certificate authorities. DSA is a
+        deprecated algorithm and should not be used.
+
+    Deserialize a public key from OpenSSH (`PROTOCOL.certkeys`_) encoded data
```

Comment:
Describing it as deserializing a public key is confusing. It's deserializing a cert... which contains a pub key.